src/Http/Form/SignupForm.php

```diff
@@ -62,7 +62,25 @@ public function validateAll(string $action = 'create'): bool
             $validSpeakerBio = $this->validateSpeakerBio();
         }
 
-        return $this->validateEmail() && $validPasswords && $this->validateFirstName() && $this->validateLastName() && $this->validateUrl() && $validSpeakerInfo && $validSpeakerBio && $this->validateSpeakerPhoto() && $agreeCoc && $this->validateJoindInUsername();
+        return $this->validateEmail() && $validPasswords &&
```

Comment:
Great idea to start wrapping here, the line has already become quite long.

Would you mind wrapping after each element?

```diff
diff --git a/src/Http/Form/SignupForm.php b/src/Http/Form/SignupForm.php
index eeea60b1..8d2630d6 100755
--- a/src/Http/Form/SignupForm.php
+++ b/src/Http/Form/SignupForm.php
@@ -62,10 +62,17 @@ class SignupForm extends Form
             $validSpeakerBio = $this->validateSpeakerBio();
         }

-        return $this->validateEmail() && $validPasswords &&
-            $this->validateFirstName() && $this->validateLastName() && $this->validateUrl() &&
-            $validSpeakerInfo && $validSpeakerBio && $this->validateSpeakerPhoto() && $agreeCoc &&
-            $this->validateJoindInUsername() && $this->validateTransportationRequests();
+        return $this->validateEmail()
+            && $validPasswords
+            && $this->validateFirstName()
+            && $this->validateLastName()
+            && $this->validateUrl()
+            && $validSpeakerInfo
+            && $validSpeakerBio
+            && $this->validateSpeakerPhoto()
+            && $agreeCoc
+            && $this->validateJoindInUsername()
+            && $this->validateTransportationRequests();
     }

     public function validateTransportationRequests()
```